addons/l10n_fr_pdp_reports/__manifest__.py

```diff
@@ -0,0 +1,28 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    "name": "France - PDP E-reporting",
+    "version": "17.0.1.0.0",
+    "category": "Accounting/Localizations/Reporting",
+    "summary": "Building blocks for France PDP e-reporting flows",
+    "author": "Odoo",
+    "depends": [
+        "base",
+        "account",
+        "account_edi",
+        "account_peppol",
+        "account_reports",
+        "l10n_fr",
+        "l10n_fr_pdp",
```

Comment:
Since we have that l10n_fr is useless, care to avoid duplicate in the depends :smile: Check the others to be sure ;)